packages/api-reference/src/components/Content/Schema/SchemaPropertyHeading.vue

```diff
@@ -81,6 +82,31 @@ const getModelNameFromSchema = (schema: OpenAPI.Document): string | null => {
 
   return null
 }
+
+/** Get the const value from the schema */
+const constValue = computed(() => {
+  if (isDefined(value?.const)) {
+    return value?.const
+  }
+
+  if (value?.enum?.length === 1) {
+    return value.enum[0]
+  }
+
+  if (value?.items) {
+    if (isDefined(value.items.const)) {
+      return value.items.const
+    }
+
+    if (value.items.enum?.length === 1) {
+      return value.items.enum[0]
+    }
+  }
+  return null
+})
+
+/** Check if the schema has a const value */
+const hasConstValue = computed(() => constValue.value !== null)
```

Comment:
```suggestion
```
This will be true for undefined! Use the trick that you taught me. I would just check it in the template
```
v-if="isDefined(constValue)"
```